### PR TITLE
added pytorch tour

### PR DESCRIPTION
**Pull request by kamil-kaczmarek**

### CODE REVIEW

File: product-tours/tour-pytorch/Tour-with-PyTorch.ipynb [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## If you want to learn more, go to the [Neptune documentation](https://docs.neptune.ai/integrations/pytorch.html)."
```
Comment (jakubczakon) [inline product-tours/tour-pytorch/Tour-with-PyTorch.ipynb:564]: I don't think this link works (as it is in the header)

File: product-tours/tour-pytorch/Tour-with-PyTorch.ipynb [hunk 2]
```diff
@@ -0,0 +1,593 @@
...
+    "colab_type": "code",
+    "id": "GvDSBSrOx-R4",
+    "outputId": "667d1b2f-cdef-44f0-f505-832205dea223",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "neptune.stop()"
```
Comment (jakubczakon) [inline product-tours/tour-pytorch/Tour-with-PyTorch.ipynb:520]: I think it is safer to run neptune.stop() before running the tests (.stop() will make all the logs from async threads synced). That said I don't think it makes a difference with this particular example and test.

File: product-tours/tour-pytorch/Tour-with-PyTorch.ipynb [hunk 3]
```diff
@@ -0,0 +1,593 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Basic example"
```
Comment (jakubczakon) [inline product-tours/tour-pytorch/Tour-with-PyTorch.ipynb:118]: I think it doesn't make sense to have a 'basic example' here as it is the only example in this notebook (atleast as I was reading it that is what I felt). If we drop it we need to update the headers with the proper number of '#'

**Review by jakubczakon — CHANGES_REQUESTED** [review]

File: product-tours/tour-pytorch/Tour-with-PyTorch.ipynb (hunk 1 above)
Comment (kamil-kaczmarek) [reply product-tours/tour-pytorch/Tour-with-PyTorch.ipynb:564]: changed to text

Comment (kamil-kaczmarek) [reply product-tours/tour-pytorch/Tour-with-PyTorch.ipynb:564]: But wait, I just checked it in Jupyter environment and it works.
Similar when I opened notebook file in the github. I think we can keep it like this.

Did yo uhave problems with the link?

File: product-tours/tour-pytorch/Tour-with-PyTorch.ipynb (hunk 2 above)
Comment (kamil-kaczmarek) [reply product-tours/tour-pytorch/Tour-with-PyTorch.ipynb:520]: ok

File: product-tours/tour-pytorch/Tour-with-PyTorch.ipynb (hunk 3 above)
Comment (kamil-kaczmarek) [reply product-tours/tour-pytorch/Tour-with-PyTorch.ipynb:118]: corrected

**Review by jakubczakon — APPROVED** [review]

**Pull request merged**